Microphone.Nancy/NancyProvider.cs

```diff
@@ -6,14 +6,13 @@
 
 namespace Microphone.Nancy
 {
-    public class NancyProvider : IFrameworkProvider
+    public class NancyProvider<TBootstrapper> : IFrameworkProvider where TBootstrapper : DefaultNancyBootstrapper
```

Comment:
Sounds good. I guess that adds a bit more flexibility if someone doesn't want to just inherit from the default bootstrapper.